### ISSUE TITLE
[Bug] pme.context.add_global() not registering any global objects, breaking panel and other commands

### ISSUE THREAD

**Issue by MichaelColina** [Pluglug/pie-menu-editor-fork#19]
## Bug Overview
The pie with a slot with a custom command such as: panel("VIEW3D_PT_tools_brush_falloff", frame=True, header=True, expand=None) appears as not available (warning sign next to the Slot name of the operand in the pie menu) and shows this tooltip when clicked; "Pie Menu Editor: NameError: name 'panel' is not defined" and for some reason it doesn't log the operation in the info editor or system console (i don't know how important is that so just mentioning in case)

## Expected Behavior
To show the expanded panel when opening the pie menu of course...

## Environment
- Blender Version: 4.3.1
- PME Version: 1.8.8
- OS: Win 10

**Comment by Pluglug** [comment]
You're right... there is no panel in PME Global...! I will look into it! Thank you so much!!!

![image](https://github.com/user-attachments/assets/0076e324-22a1-4279-bec3-558fa0d3c0fb)

**Comment by Pluglug** [comment]
Hey @MichaelColina, I've made the fix, could you try this build for now?

https://github.com/Pluglug/pie-menu-editor-fork/archive/refs/heads/4.3-compat.zip

I think you're the only one using 1.18.8a... lol Thanks so much!

**Comment by Pluglug** [comment]
Note: Currently `U` and `custom_icon` are not registered. We will continue to investigate.
